### ISSUE TITLE
Accessible p-treeTable: reach and identifie nodes by keyboard or screenreaders, keep focus

### ISSUE THREAD

**Issue by moma1806** [primefaces/primeng#10393]
[x ] feature request 

**Current behavior**
As blind user I must reach the control elements to open and close the level of a tree table data presentation with the keyboard. The respectively level should be read out by screenreaders. In order to know where I am located on the page after opening or closing a level, the control element should be read out after it was operated.

**Expected behavior**
**Acceptance criteria: Accessible p-treeTable** 

**Tabbable closing or opening node**
**Given** A p-treeTable and Jaws 2021 and Google Chrome. **Then** the nodes to open or close a level should be tabbable by keyboard (no tabindex -1) 

**Individual title identifier**
**When** the nodes to open or close a level are tabbed **then** the identifier of the node should be read by the screenreader **and** the node must have an individual custom identifier title for the level Example "Open Economics Faculty" 

**Mouse over text**
**When** the mouse pointer is moved over a node button to open or close a level **then** a popup with the labelling of the node is displayed (title-attribute) 

**Focus after closing or opening node**
**When** the node to open or close a level is clicked, **then** the focus should stay on the node **and** the identifier of the node should be read out again by the screenreader **Example** "Close Economics Faculty" **and** the focus on the node should be graphically perceptible. 

**Reach next control element by tab**
**When** the node to open or close a level is clicked, **and then** the tabkey is clicked, **then** the screenreader reads out the next control element in the treetable. 

**Minimal reproduction of the problem with instructions**
https://www.primefaces.org/primeng/showcase/#/treetable/templating

**What is the motivation / use case for changing the behavior?**
Web Content Accessibility Guidelines (WCAG) 2.1 requirements 
   1.1.1 Non-text Content  
   1.3.1 Info and Relationships
   2.1.1 Keyboard    
   2.4.3 Focus Order   
   2.4.6 Headings and Labels

**Please tell us about your environment:**
Windows, Linux, Android, iOS... IDE: Eclipse, HTTP Server: Apache Tomcat
* **Angular version:** 11.1.4
* **PrimeNG version:** primeng 11.0.0
* **Browser:** Chrome, Firefox, Edge, Safari, Safari on IOS, Chrome on Android; all browsers in newest version
* **Language:** typescript  4.1.3
* **Node (for AoT issues):** node --version 12.5.0

**Comment by cetincakiroglu** [comment]
Hi,

So sorry for the delayed response! Improvements have been made to many components recently, both in terms of performance and enhancement. Therefore, this improvement may have been developed in another issue ticket without realizing it. You can check this in the documentation. If there is no improvement on this, can you reopen the issue so we can include it in our roadmap? 
Please don't forget to add your feedback as a comment after reopening the issue. These will be taken into account by us and will contribute to the development of this feature. Thanks a lot for your understanding!

Best Regards,
